src/elements/common/messages.js

```diff
@@ -980,6 +980,31 @@ const messages = defineMessages({
         description: 'Label for a button that displays more options',
         defaultMessage: 'More options',
     },
+    bookmark: {
+        id: 'be.bookmark',
+        description: 'Icon title for a Box item of type bookmark or web-link',
+        defaultMessage: 'Bookmark',
+    },
+    collaboratedFolder: {
+        id: 'be.collaboratedFolder',
+        description: 'Icon title for a Box item of type folder that has collaborators',
+        defaultMessage: 'Collaborated Folder',
+    },
+    externalFolder: {
+        id: 'be.externalFolder',
+        description: "Icon title for a Box item of type folder that has collaborators outside of the user's enterprise",
```

Comment:
```suggestion
        description: 'Icon title for a Box item of type folder that has collaborators outside of the user's enterprise',
```

Reply:
I think you'll need to escape the apostrophe

Reply:
Prettier seems to complain 😿 
<img width="902" alt="image" src="https://user-images.githubusercontent.com/34158284/216444524-500e4ee5-54de-4b1e-a24e-9d48a52461f1.png">

Reply:
Ah ok!